.travis.yml

```diff
@@ -1,24 +1,27 @@
+sudo: false
 language: python
 
+env:
+  global:
+    PYTEST_ADDOPTS="--cov vint --cov-report=xml --cov-report=term-missing:skip-covered"
+
 python:
+  - "2.7"
   - "3.4"
-
-env:
-  - TOXENV="py27"
-  - TOXENV="py34"
-  - TOXENV="coverage"
+  - "3.5"
+  - "3.6"
 
 install:
-  - pip install tox
-  - if test "$TOXENV" = "coverage"; then pip install coveralls; fi
+  - pip install tox tox-travis pytest-cov
```

Comment:
pytest-cov would not be necessary here.  But could maybe get removed from testing requirements?!